test/ASTGen/diagnostics.swift

```diff
@@ -66,3 +71,12 @@ func misisngPatternTest(arr: [Int]) {
   for {} // expected-error {{expected pattern, 'in', and expression in 'for' statement}}
          // expected-note@-1 {{insert pattern, 'in', and expression}} {{7-7=<#pattern#> }} {{7-7=in }} {{7-7=<#expression#> }}
 }
+
+using @MainActor // expected-note {{default isolation was previously declared here}}
+using nonisolated // expected-error {{invalid redeclaration of file-level default actor isolation}}
```

Comment:
Hah, exactly what I was worried about when I saw the proposal update :)
Good to see we diagnose this.